api/Selection.json

```diff
@@ -0,0 +1,287 @@
+{
+  "api": {
+    "Selection": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection#Browser_compatibility",
+        "support": {
+          "chrome": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "The <a href='https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onselectionchange'><code>GlobalEventHandlers.onselectionchange</code></a> and <a href='https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onselectstart'><code>GlobalEventHandlers.onselectstart</code></a> event handlers are supported as of Firefox 52."
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": true
+          },
+          "webview_android": {
+            "version_added": null
```

Comment:
This is `true`. In this file, `webview_android` should match `chrome` and `chrome_android`.